pipeline_tools/shared/submission/get_analysis_workflow_metadata.py

```diff
@@ -86,6 +97,8 @@ def main():
     parser = argparse.ArgumentParser()
     parser.add_argument('--analysis_output_path', required=True)
     parser.add_argument('--cromwell_url', required=True)
+    parser.add_argument('--include_keys', required=False, nargs='+')
+    parser.add_argument('--include_subworkflows', default="False", choices=["True", "False"])
```

Comment:
```suggestion
    parser.add_argument('--include_subworkflows', default=False)
```
Change to boolean rather than string